components/searchwithin/stories/searchwithin.stories.js

```diff
@@ -1,11 +1,11 @@
 // Import the component markup template
-import { Template } from "./template";
+import { Template } from "@spectrum-css/searchwithin/stories/template.js";
 
 import { default as PickerStories } from "@spectrum-css/picker/stories/picker.stories.js";
 const ignoreProps = ["rootClass", "position", "isRounded"];
 
 export default {
-	title: "Components/Search within",
+	title: "Deprecated/Search within",
 	description: "The Search within component is...",
```

Comment:
```suggestion
```

Since we're closing out maintenance of this should we just delete the boilerplate description?